### PR TITLE
Chore/bump runtime

### PR DESCRIPTION
**Pull request by jkoenig134**
Changes from https://github.com/nmshd/runtime/compare/6.0.1...6.1.0

### CODE REVIEW

File: packages/enmeshed_types/lib/src/dtos/token.dart
Comment (Magnus-Kuhn) [inline packages/enmeshed_types/lib/src/dtos/token.dart:1]: `token_test.dart` could test the new addition

**Review by Magnus-Kuhn — COMMENTED** [review]
What's the purpose of the `library` keyword? The documentation is unclear about this.

**Comment by jkoenig134** [comment]
> What's the purpose of the `library` keyword? The documentation is unclear about this.

please ignore this :)

**Comment by jkoenig134** [comment]
@Siolto how was you test about the datawallet bugfix?

**Review by Magnus-Kuhn — APPROVED** [review]

**Comment by Milena-Czierlinski** [comment]
@jkoenig134 What are the connector sdk endpoints used for here in the app? I noticed that you updated them, but not the actual facades.

**Pull request merged**